### PR TITLE
Issue 294

### PR DESCRIPTION
**Pull request by josebarrueta**

### CODE REVIEW

File: api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java [hunk 1]
```diff
@@ -130,6 +130,50 @@
     IdSiteUrlBuilder setShowOrganizationField(boolean showOrganizationField);
 
     /**
+     * Sets the {@code sp_token} property to be used during the initial sso call. The {@code sp_token} must correspond to
+     * an actual password reset token and when successful the {@code IdSite} will be redirected to the IdSite.
+     * <p>
+     * <h5>Example</h5>
+     * <p>
+     * <p>Assume your ID Site is located at the domain {@code id.myapp.com}.  If you specify an
+     * {@link #setSpToken(String) spToken} of {@code aSpToken} and set
```
Comment (mrioan) [inline api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java]: the `aSpToken` value was introduced in the explanation but was not used in the example url...

File: api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java [hunk 2]
```diff
@@ -119,7 +119,7 @@
      * the end-user in the ID Site user interface.
      * <p>
      * <p>Setting this to {@code true} allows the user to see the field and potentially change the value.  This might be
-     * userful if users can have accounts in different organizations - it would allow the user to specify which
+     * useful if users can have accounts in different organizations - it would allow the user to specify which
```
Comment (mrioan) [inline api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java]: ...it *wil* allow the user...

File: api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java [hunk 3]
```diff
@@ -130,6 +130,50 @@
     IdSiteUrlBuilder setShowOrganizationField(boolean showOrganizationField);
 
     /**
+     * Sets the {@code sp_token} property to be used during the initial sso call. The {@code sp_token} must correspond to
+     * an actual password reset token and when successful the {@code IdSite} will be redirected to the IdSite.
```
Comment (mrioan) [inline api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java]: not sure what this sentence means: "when successful the {@code IdSite} will be redirected to the IdSite"

File: api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java [hunk 4]
```diff
@@ -130,6 +130,50 @@
...
+     * <h5>Example</h5>
+     * <p>
+     * <p>Assume your ID Site is located at the domain {@code id.myapp.com}.  If you specify an
+     * {@link #setSpToken(String) spToken} of {@code aSpToken} and set
+     * {@link #setPath(String) path} to {@code #reset}, the user will be sent to
+     * {@code https://id.myapp.com/#reset?jwt=....} to complete the reset password process in your {@code IdSite}.
+     *
+     * @param spToken a unique token used to reset a password.
+     * @return this instance for method chaining.
+     * @since 1.0.RC5
```
Comment (mrioan) [inline api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java]: Version is now 1.0.RC5.1

**Comment by mrioan** [comment]
Back to you @josebarrueta. I finished the review. If you are busy I think I can update the PR myself based on my comments. Let me know and I will be happy to help... Thanks!

File: api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java (hunk 2 above)
Comment (josebarrueta) [inline api/src/main/java/com/stormpath/sdk/idsite/IdSiteUrlBuilder.java]: Hi @mrioan, in this line I just corrected the typo in `userful`, furthermore I think `would` is OK.

**Comment by josebarrueta** [comment]
@mrioan JavaDoc has been updated. Thanks!

**Comment by mrioan** [comment]
Approved. Merging.

**Pull request merged**